pallets/msa/src/benchmarking.rs

```diff
@@ -205,11 +198,13 @@ benchmarks! {
 	on_initialize {
 		// we should not need to max out storage for this benchmark, see:
 		// https://substrate.stackexchange.com/a/4430/2060
-		let m in 1 .. 3_000;
+		let m in 1 .. T::MaxSignaturesPerBucket::get();
 		for j in 0 .. m {
 			let mortality_block = 49 as u32;
```

Comment:
what is the reason behind keeping `mortality_block` fixed?

Reply:
Keeping it constant makes it easy since making mortality_block more dynamic would not impact the benchmarks.